src/Toolkit/Toolkit.WinUI/UI/Controls/FeatureForm/FeatureForm.Theme.xaml

```diff
@@ -0,0 +1,326 @@
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="IsTabStop" Value="False" />
+    </Style>
+
+    <Style TargetType="primitives:FieldFormElementView" >
+        <Setter Property="Margin" Value="{StaticResource FeatureFormElementInputMargin}" />
+        <Setter Property="ComboBoxFormInputTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <primitives:ComboBoxFormInputView Element="{Binding}" IsEnabled="{Binding IsEditable}" />
+                </DataTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="RadioButtonsFormInputTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <primitives:RadioButtonsFormInputView Element="{Binding}" IsEnabled="{Binding IsEditable}" />
+                </DataTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="SwitchFormInputTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <primitives:SwitchFormInputView Element="{Binding}" IsEnabled="{Binding IsEditable}" />
+                </DataTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="TextAreaFormInputTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <primitives:TextFormInputView Element="{Binding}" MaxLines="4" MinLines="4" IsEnabled="{Binding IsEditable}" />
+                </DataTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="TextBoxFormInputTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <primitives:TextFormInputView Element="{Binding}" IsEnabled="{Binding IsEditable}" />
+                </DataTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="BarcodeScannerFormInputTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <primitives:TextFormInputView Element="{Binding}" IsEnabled="{Binding IsEditable}" />
+                </DataTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="DateTimePickerFormInputTemplate">
+            <Setter.Value>
+                <DataTemplate>
+                    <primitives:DateTimePickerFormInputView Element="{Binding}" IsEnabled="{Binding IsEditable}" />
+                </DataTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="primitives:FieldFormElementView">
+                    <StackPanel>
+                        <TextBlock Text="{Binding Label}" Style="{StaticResource FeatureFormViewTitleStyle}"/>
+                        <ContentControl Margin="0,3" Foreground="Gray" Content="{Binding}" x:Name="FieldInput" IsTabStop="False" ToolTipService.ToolTip="{Binding Hint}" HorizontalContentAlignment="Stretch" />
```

Comment:
I am seeing small empty tooltips pop up in the sample form:

![2024-08-02_000115 Toolkit SampleApp WinUI](https://github.com/user-attachments/assets/71e90b67-fe5d-412e-b762-36f9d3818865)

Is the text missing, or is this a case of an empty-string / all-whitespace hint that should be hidden?